src/components/DataSubmissions/EditSubmissionNameDialog.tsx

```diff
@@ -0,0 +1,242 @@
+import { useMutation } from "@apollo/client";
+import { LoadingButton } from "@mui/lab";
+import { Box, Typography, styled } from "@mui/material";
+import DialogActions from "@mui/material/DialogActions";
+import { useSnackbar } from "notistack";
+import React, { useState, useEffect } from "react";
+
+import CloseIconSvg from "../../assets/icons/close_icon.svg?react";
+import {
+  mutation as EDIT_SUBMISSION,
+  Input as EditSubmissionInput,
+  Response as EditSubmissionResponse,
+} from "../../graphql/updateSubmissionName";
+import DefaultDialog from "../StyledDialogComponents/StyledDialog";
+import StyledCloseDialogButton from "../StyledDialogComponents/StyledDialogCloseButton";
+import DefaultDialogContent from "../StyledDialogComponents/StyledDialogContent";
+import DefaultDialogHeader from "../StyledDialogComponents/StyledHeader";
+import StyledHelperText from "../StyledFormComponents/StyledHelperText";
+import StyledLabel from "../StyledFormComponents/StyledLabel";
+import StyledOutlinedInput from "../StyledFormComponents/StyledOutlinedInput";
+
+const StyledDialog = styled(DefaultDialog)({
+  "& .MuiDialog-paper": {
+    width: "577px !important",
+    height: "402.86px",
+    borderRadius: "8px",
+    borderWidth: "2px",
+    borderStyle: "solid",
+    borderColor: "#5AB8FF",
+    paddingTop: "28px",
+    paddingRight: "34px",
+    paddingBottom: "60px",
+    paddingLeft: "34px",
+    boxSizing: "border-box",
+    background: "#F2F6FA",
+    boxShadow: "0px 4px 45px 0px rgba(0,0,0,0.4)",
+  },
+});
+
+const StyledTitle = styled(Typography)({
+  fontFamily: "Nunito, 'Rubik', sans-serif",
+  fontWeight: 600,
+  fontSize: "13px",
+  lineHeight: "27px",
+  letterSpacing: "0.5px",
+  textTransform: "uppercase",
+  color: "#929292",
+});
+
+const StyledDialogActions = styled(DialogActions)({
+  padding: "0 !important",
+  justifyContent: "center",
+  marginTop: "20px !important",
+  gap: "10px",
+});
+
+const baseButtonStyles: object = {
+  width: "128px !important",
+  height: "50.59px",
+  padding: "12px 60px",
+  borderRadius: "8px",
+  textAlign: "center",
+  fontFamily: "'Nunito', 'Rubik', sans-serif",
+  fontWeight: 700,
+  fontSize: "16px",
+  letterSpacing: "0.32px",
+  textTransform: "none" as const,
+  margin: "0 15px",
+  "&:hover": {
+    background: "transparent",
+  },
+};
+
+const StyledButton = styled(LoadingButton)({
+  ...baseButtonStyles,
+  background: "#FFF",
+  border: "1px solid #000",
+  color: "#000",
+});
+
+const StyledGreenButton = styled(LoadingButton)({
+  ...baseButtonStyles,
+  color: "#fff",
+  borderColor: "#26B893",
+  background: "#1B8369",
+});
```

Comment:
For this particular button, we do have a theme that will fix some of the styling issues.

1. Remove the `&:hover` styling from the baseButtonStyles
2. Apply the suggestion below to the `StyledGreenButton` variable
3. Add the prop `color="success"` to the `<StyledGreenButton>` component

```suggestion
const StyledGreenButton = styled(LoadingButton)({
  ...baseButtonStyles,
});
```